index.js

```diff
@@ -19,9 +19,31 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    await Recipe.create({
+      title: "Hot Dog Fingers",
+      level: "UltraPro Chef",
+      ingredients: ["5 hot dogs", "5 human fingers", "100g of cut off nails"],
+      cuisine: "Anthropofagic",
+      dishType: "snack",
+      duration: 120,
+      creator: "Chef Hannibal",
+    });
+    console.log("Hot Dog Fingers");
```

Comment:
Here you are console logging the title hard coded. The purpose of this step was to console.log the title no meter what is the recipe that you create. One way of doing this is storing the Recipe.create in a variable:

```
const oneRecipe = await Recipe.create({
      title: "Hot Dog Fingers",
      level: "UltraPro Chef",
      ingredients: ["5 hot dogs", "5 human fingers", "100g of cut off nails"],
      cuisine: "Anthropofagic",
      dishType: "snack",
      duration: 120,
      creator: "Chef Hannibal",
    });
    console.log(`New recipe added: ${oneRecipe.title}`);
```